AudioKit/Common/Internals/Audio File/AKAudioFile+Async Processing.swift

```diff
@@ -0,0 +1,365 @@
+//
+//  AKAudioFile+processing Async.swift
```

Comment:
I didn’t know that…

Sorry :-/

Laurent.


> Le 14 juil. 2016 à 15:57, Aurelius Prochazka <notifications@github.com> a écrit :
> 
> In AudioKit/Common/Internals/Audio File/AKAudioFile+Async Processing.swift <https://github.com/audiokit/AudioKit/pull/476#discussion_r70808869>:
> 
> > @@ -0,0 +1,365 @@
> > +//
> > +//  AKAudioFile+processing Async.swift
> This title doesn't match the filename. Also, there should be no spaces in filenames.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/audiokit/AudioKit/pull/476/files/23c093dd84af517b2f69e8a3c258d7469e225982#r70808869>, or mute the thread <https://github.com/notifications/unsubscribe/ASTVrmgd3Eh0KItSoHZZ6aprfLrUXl8Rks5qVkA3gaJpZM4JMY_q>.
> 




Mes compos en téléchargement libre sont sur Soundcloud (musicatalogue) :

https://soundcloud.com/musicatalogue